test/lib/usd/translators/UsdImportRfMShadersTest/MarbleCube.usda

```diff
@@ -0,0 +1,73 @@
+            float3 xformOp:translate:pivot = (0, 0, 2.5)
+            uniform token[] xformOpOrder = ["xformOp:translate:pivot", "!invert!xformOp:translate:pivot"]
+        }
+    }
+
+    def Scope "Materials"
+    {
+        def Material "MarbleCubeSG"
+        {
+            # XXX: Note that the rendermanForMaya material conversion at export
+            # will create this surface terminal output using the "ri"
+            # renderContext as opposed to using the universal renderContext as
+            # we do here. Further work is needed in the "useRegistry"
+            # shadingMode to be able to support finding materials in
+            # renderContexts other than universal.
+            token outputs:surface.connect = </MarbleCube/Materials/MarbleCubeSG/MarbleLambert.outputs:surface>
```

Comment:
Here's where we're cheating for now. If you were to run the export side of this test, it would author the terminal using the `ri` renderContext:

```
token outputs:ri:surface.connect = ...
```

But right now, the `useRegistry` shadingMode doesn't know to look anywhere other than the universal renderContext when looking for shaders to import. I'll revisit this to use the correct renderContext once we add that support.